linclogger/log_formatter.py

```diff
@@ -28,15 +31,19 @@ def process_log_record(self, log_record):
         if self._extra is not None:
             for key, value in self._extra.items():
                 log_record[key] = value
-        errno = None
         if log_record['levelname'] in (logging.ERROR, logging.CRITICAL):
-            try:
-                errno = zlib.adler32(log_record['message'])
-            except zlib.error:
-                log.warn('Zlib cant calculate format string: %s checksum', log_record['message'])
-            except Exception as e:
-                log.warn('Fails to generate errno: %s', e)
-        log_record['errno'] = errno if errno is not None else ''
+            # genereate a checksum for the error
+            if 'message' in log_record:
+                errno = None
+                try:
+                    message = str(log_record['message'])
+                    errno = zlib.adler32(message.encode('utf-8')) & 0xffffffff
+                except zlib.error:
+                    log.warn('Zlib cant calculate format string: %s checksum', log_record['message'])
```

Comment:
[very minor] since log is at a global level, better to rename it as `LOG` or `LOGGER`. Just a suggestion